### DISCUSSION TITLE
linux service generation

### DISCUSSION THREAD

**Discussion by seagullmike** [sbt/sbt-native-packager#1423] [Q&A]
I am trying to understand the process to generate a linux service based on scripts generated from the packager.
I have found the SystemdPlugin I put it in my build.sbt but when I run universal:packageXzTarball I get a package, but there is not a service file generated.  I find the docs to customize the service template, but I don't know what the template should look like when it is to call a service based on scripts generated from JavaServerAppPackaging

is there a task that generates the service file?

**Comment by muuki88** [comment]
Hi,

Thanks for your question. Can you give a minimal `build.sbt` to reproduce?

Did you enable the `JavaServerAppPackaging` plugin?
